forms/serializers/form.py

```diff
@@ -245,20 +239,77 @@ class Meta:
             ),
         ]
 
+    def entry_generator(self, entries, sections=None, fields=None):
+        if "sections" in entries:
+            yield from self.entry_generator(entries["sections"], sections=[entry for entry in entries["sections"]])
+        if "fields" in entries:
+            yield from self.entry_generator(entries["fields"], sections=sections, fields=[entry for entry in entries["fields"]])
```

Comment:
And the same thing happening here with the fields. Maybe I should watch this closer by truly debugging/testing this function.